backend/app/services/attachment/attachment_service.py

```
@@ -215,35 +239,88 @@ def delete_attachment(
+        Raises:
+            ValueError: If binary data cannot be retrieved
+        """
+        # Check if data is stored in MinIO (new data)
+        if attachment.minio_object_key:
+            try:
+                data = minio_client.download_file(attachment.minio_object_key)
+                logger.debug(
+                    f"Retrieved attachment {attachment.id} from MinIO: "
+                    f"{attachment.minio_object_key} ({len(data)} bytes)"
+                )
+                return data
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)